test/test_foreach.py

```diff
@@ -126,6 +126,7 @@ def _test_bin_op_list_alpha(self, device, dtype, foreach_op, foreach_op_, torch_
             else:
                 self.assertEqual(tensors1, expected)
 
+    @skipMeta
```

Comment:
I don't remember the details, but without this I saw a bunch of messages which seem like a ton of failures, but they were somehow treated as warnings or something like that.
The reason I added this decorator is, those messages were just saying "hey this operation is not working with meta tensor" but coming with the bunch of redundant information (at least for foreach tests).

Reply:
It that's the case (the tests are actually passing), don't skip them please, so that if coverage is extended to meta they don't have to be explicitly reenabled. For you local runs you could still filter to only CPU or only CUDA device (not both together I'm afraid)

Reply:
Yes, please don't do this. If you want to reduce the spamminess, edit the code in torch/testing/_internal/common_utils.py

```
@contextmanager
def skip_exception_type(exc_type):
    try:
        yield
    except exc_type as e:
        raise unittest.SkipTest(f"not implemented: {e}") from e
```

you could make it stop printing the entire exception text (though I've personally found it useful for collecting information from XML logs)

Reply:
Sure, thank you @ezyang for the explanation